vendor/src/github.com/docker/engine-api/types/container/config.go

```diff
@@ -24,6 +38,7 @@ type Config struct {
 	StdinOnce       bool                  // If true, close stdin after the 1 attached client disconnects.
 	Env             []string              // List of environment variable to set in the container
 	Cmd             strslice.StrSlice     // Command to run when starting the container
+	Healthcheck     *HealthConfig         `json:",omitempty"` // Command to run to check container is healthy
```

Comment:
You could with a script, but I guess just using curl as your healthcheck is quite nice.